static/js/common.js

```diff
@@ -37,13 +37,14 @@ exports.password_quality = function (password, bar, password_field) {
 
     // Compute a quality score in [0,1].
```

Comment:
This comment is getting out of touch with what the code is doing -- let's just delete it. The code itself isn't doing anything really complicated, so it should be as clear as a comment anyway.